scripts/background.js

```diff
@@ -1,3 +1,79 @@
+function setStartDate(){
+  const currentTime = (new Date()).toJSON();
+  const items = { 'start_date': currentTime }; 
+  chrome.storage.local.set(items, () => {
+      if (chrome.runtime.lastError) {
+          console.error(chrome.runtime.lastError.message);
+      }
+      else{
+        console.log('set start_date:' + currentTime);
```

Comment:
are these `console.log` necessary for prod?

Reply:
Not at all. I added it because I couldn't find a way to notice all the changes.